documenting MC client connections when member security is enabled

erosb opened the pull request:
Backport of #50 to 4.y

Serdaro commented on docs/modules/ROOT/pages/connecting-members.adoc:
```diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
+
+== Configuring Security on the Members
+
+NOTE: this section applies only to IMDG 4.2 and newer. If you use an older version, you can skip this configuration step.
```
NOTE: This section ...

Serdaro commented on docs/modules/ROOT/pages/connecting-members.adoc:
```diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
+
+NOTE: this section applies only to IMDG 4.2 and newer. If you use an older version, you can skip this configuration step.
+
+If `security` is enabled in the member configurations, then the `management-permission` should be granted for the MC client. Also, in this case the client should authenticate itself, which means you need to set up the connection by uploading a configuration XMl (or YAML) file (see above).
```
... granted for the Management Center client. ...

... a configuration file (XML or YAML); see xref:ROOT:managing-clusters.adoc#creating-a-cluster-configuration-by-uploading-file[here].

Serdaro commented on docs/modules/ROOT/pages/connecting-members.adoc:
```diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
+
+If `security` is enabled in the member configurations, then the `management-permission` should be granted for the MC client. Also, in this case the client should authenticate itself, which means you need to set up the connection by uploading a configuration XMl (or YAML) file (see above).
+
+Example member configuration:
```
An example member configuration is shown below:

Serdaro commented on docs/modules/ROOT/pages/connecting-members.adoc:
````diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
+
+```
+
+Example client configuration:
````
An example client configuration is shown below:

Serdaro reviewed (commented):
Minor comments.

erosb commented:
Thank you for the review @Serdaro , I addressed your comments.
The same PR (#50) was already merged into `main`, so I will create a new PR addressing your comments there as well.
